doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -92,3 +92,130 @@ Talk to the Lab Owners if special hardware or topology requirements are needed,
 **Lab Quantities and Duration of Lab Use**
 
 There is no limit to the number of labs which can be secured, or the duration.  The limiting factor is resource available.  Lab Owners reserve the right to reclaim underutilized labs and reassign to other teams where there is demand for labs.  It is important that the Lab User have a schedule and plan in place to utilize the labs to retain the lab for testing.
+
+<a name="4.5"></a>
+## 4.5 OPNFV Pharos Project
+
+For this RI, the OPNFV Pharos specification has been chosen as labs with pods of servers already exist under OPNFV's
+Lab as a Service (LaaS).
+
+The Pharos Project deals with developing an OPNFV lab infrastructure that is geographically and technically diverse. This
+will greatly assist in developing a highly robust and stable OPNFV platform. Community labs are hosted by individual
+companies and there is also an OPNFV lab hosted by the Linux Foundation that has controlled access for key development
+and production activities. The Pharos Specification defines a “compliant” deployment and test environment. Pharos is
+responsible for defining lab capabilities, developing management/usage policies and process; and a support plan for
+reliable access to project and release resources. Community labs are provided as a service by companies and are not
+controlled by Pharos however our goal is to provide easy visibility of all lab capabilities and their usage at all-times.
+
+A requirement of Pharos labs is to provide bare-metal for development, deployment and testing. This is resource intensive
+from a hardware and support perspective while providing remote access can also be very challenging due to corporate IT
+policies. Achieving a consistent look and feel of a federated lab infrastructure continues to be an objective. Virtual
+environments are also useful and provided by some labs. Jira is currently used for tracking lab operational issues as
+well as for Pharos project activities.
+
+<a name="4.5.1"></a>
+### 4.5.1 Pharos Specification
+
+The Pharos Specification defines a hardware environment for deployment and testing of the OPNFV platform release.
+
+Pharos lab infrastructure has the following objectives:
+
+* Provides secure, scalable, standard and HA environments for feature development
+* Supports the full Euphrates deployment lifecycle (this requires a bare-metal environment)
+* Supports functional and performance testing of the Euphrates release
+* Provides mechanisms and procedures for secure remote access to Pharos compliant environments for OPNFV community
+* Deploying OpenStack in a Virtualized environment is possible and will be useful, however it does not provide a fully
+  featured deployment and realistic test environment for the Euphrates release of OPNFV.
+
+The high level architecture is outlined in the following diagram:
```

Comment:
@markbeierl The networks defined in the two diagrams (not numbered: ch04_opnfv_lab_topology and ch04_opnfv_pharos_high_level) in RI 4.5.1 should be named consistently. It would also be nice to map them to the networks defined in RA-1 Ch04: [4.2.3.2 High Level Logical Network Layout](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4232-high-level-logical-network-layout).